src/main/java/org/folio/edge/api/utils/util/PropertiesUtil.java

```diff
@@ -0,0 +1,50 @@
+package org.folio.edge.api.utils.util;
+
+import static java.lang.String.format;
+
+import java.io.FileInputStream;
+import java.io.InputStream;
+import java.lang.invoke.MethodHandles;
+import java.net.URL;
+import java.util.Properties;
+import java.util.regex.Pattern;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.folio.edge.api.utils.exception.AuthorizationException;
+
+public class PropertiesUtil {
+
+  private static final Logger LOGGER = LogManager.getLogger(MethodHandles.lookup().lookupClass());
```

Comment:
For consistency with other classes in this library
```suggestion
  private static final Logger logger = LogManager.getLogger(PropertiesUtil.class);
```

Reply:
updated